source/extensions/outlier_detection_monitors/common/BUILD

```diff
@@ -0,0 +1,25 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_extension_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_extension_package()
+
+envoy_cc_extension(
```

Comment:
This is not an extension and please use `envoy_cc_library` with appropriate visibility. Then, you can remove the redundant `"envoy.outlier_detection_monitors.common"` from the `extensions_build_config.bzl`.

Reply:
Ah, I was looking so long for a way to not to put it as an extension into *.bzl files and the solution was so easy. Thanks a lot!

Reply:
Eh, it seems that does not work after all. After making it a envoy_cc_library and removing "envoy.outlier_detection_monitors.common" from the extensions_*.bzl I cannot execute tests in source/extensions/outlier_detection_monitors/common/.... It says:

`ERROR: No test targets were found, yet testing was requested`

As I said, I spent lots of time on this problem and there was always something failing. Docs, tests, etc. The only way I could make it working was to have "envoy.outlier_detection_monitors.common" in `extensions_*.bzl`

Reply:
Nevermind. I think I managed to do it! `source/extensions/outlier_detection_monitors/common` is not extension anymore.